### PR TITLE
EF of the Shisheido products added

### PR DESCRIPTION
**Pull request by deliceli**
**Is this a new addition or an update to an existing factor?**

Update

**What category or categories does it affect?**

Personal Care and Accessories

**What?**

EF's of some Shiseido products added to the OEFDB

**Why?**

Client request

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -19079,3 +19079,12 @@ a2db0d60-2494-43d7-ac89-0c71a42a677d,Energy,Electricity,electricity-energy_sourc
...
 51711eba-7a91-4f13-9f0a-08fe7e49f0df,Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kWh,not-supplied,not-supplied,0.1,1.7e-05,2e-06,not-supplied,not-supplied,,2,electricity_generation,UNFCCC,,2021,2021,2019,CA-YT,,Climatiq,2021/07/30,Emission intensity of grid mix as reported. Published in Canada 2021 National Inventory Report (NIR). Includes generation factor.,https://unfccc.int/documents/271493
+,Consumer Goods and Services,Personal Care and Accessories,consumer_goods-type_facial_wash,,g,not-supplied,0.041,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,cradle_to_shelf,Shiseido,,2021,2021,,GLOBAL,,Climatiq,2022/11/25,Emission intensity of facial wash consumer good.Factors retrieved from the Shiseido's Environmental Data Set.Shiseido observes environmental laws and regulations thoroughly by conducting compliance evaluation based on ISO14001 standards.,https://corp.shiseido.com/en/sustainability/performance/env/
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:19082]: I think lca_activity should be cradle_to_grave, wdyt @maryarbab  ?

[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:19082]: I suggest adjusting the description to: "Emission intensity of facial wash. Factors were obtained from the Environmental Product Footprint LCA-based study conducted by Shiseido according to ISO 140401."

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:19082]: @AnastasiaLobanova Agree that it is full LCA, i.e. cradle_to_grave

[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:19082]: Not sure if it is only scope 3. My suggestion is to leave the scope field blank

[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:19082]: Also agree with rewording the description.

[the remark this answers is not in the document]
Comment (deliceli) [reply OpenEmissionFactorsDB.csv:19082]: Changed everything.

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Nice work Egeberk! We need to change now the schema.toml file in the metadata folder - to add new units, as well as new LCA activity values.

**Comment by deliceli** [comment]
> 
Screenshot 2022-12-02 at 11 54 40

Should i add it manually from that button?

**Comment by AnastasiaLobanova** [comment]
If it is possible then why not. I always do it from my local version using Visual Studio Code

**Review by AnastasiaLobanova — APPROVED** [review]
Looks good to me!

**Review by maryarbab — APPROVED** [review]

**Pull request merged**